tools/sendinblueAPIPlayground.py

```diff
@@ -0,0 +1,24 @@
+import os
+import requests
+from pprint import pprint
+
+listes_ecosante_folder_name = 'listes_ecosante'
+
+# Start with
+# SENDINBLUE_API_KEY=<key> python3 tools/sendinblueAPIPlayground.py 
+
+if __name__ == "__main__":
+    api_key = os.environ['SENDINBLUE_API_KEY']
+    pprint('API key: {}'.format(api_key))
+    sendinblue_default_headers = {"accept": "application/json", "api-key": api_key}
+
+    r = requests.get('https://api.sendinblue.com/v3/contacts/folders', headers=sendinblue_default_headers)
+    pprint(r.status_code)
```

Comment:
you may want to look at `r.raise_for_status()` it will throw an exception if the status code is not ok